compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -269,9 +266,9 @@ internal class ComposeSceneMediator(
     /**
      * View wrapping the hierarchy managed by this Mediator.
      */
-    private val view = UIKitTransparentContainerView(
-        onLayoutSubviews = ::updateLayout
-    )
+    private val overlayView = UIKitTransparentContainerView(onLayoutSubviews = ::updateLayout)
+
+    fun getOverlayView(): UIView = overlayView
```

Comment:
It's a pity that the reference should be exposed outside of this class, but anyway I guess it should remain as property syntax. 

```suggestion
    private val _overlayView = UIKitTransparentContainerView(onLayoutSubviews = ::updateLayout)
    val overlayView: UIView get() = overlayView
```

Reply:
Done